.github/workflows/docs.yml

```diff
@@ -0,0 +1,34 @@
+name: Rust Docs
```

Comment:
AFAIU this job needs to be removed.

Reply:
Thanks @rcny, I have just remove it

Reply:
@rcny can you please remove the change request?